js/src/common/helpers/icon.tsx

```diff
@@ -1,11 +1,13 @@
+import * as Mithril from 'mithril';
+
 /**
  * The `icon` helper displays an icon.
  *
  * @param {String} fontClass The full icon class, prefix and the icon’s name.
- * @param {Object} attrs Any other attributes to apply.
```

Comment:
Should we also remove repeated types from JSDoc? I think they can be inferred from Typescript types.